evm_arithmetization/src/cpu/kernel/asm/beacon_roots.asm

```diff
@@ -0,0 +1,47 @@
+/// EIP-4788: Beacon block root in the EVM
```

Comment:
Add link to the pseudocode section in the EIP? https://eips.ethereum.org/EIPS/eip-4788#pseudocode